### ISSUE TITLE
V2 schema auspice config feedback

### ISSUE THREAD

**Issue by trvrb** [nextstrain/augur#344]
@emmahodcroft @jameshadfield ---

I just updated `auspice_config.json` in `nextstrain/zika` based on deprecation warnings: https://github.com/nextstrain/zika/commit/0d447cafaef77210c75c4a9f251fdef65b9e8cf2. Thanks for making it super clear what needed to happen to upgrade. Much appreciated. I do have just a couple notes of feedback on the resulting `auspice_config.json`.

1. `geographic_info`: Previously, this was `geo`. This really means something like "display these geographic resolutions on the map" and actually affects the "Geographic resolution" dropdown in auspice. How about relabeling this to `geo_resolutions`? This would more neatly correspond to intention and also to the resulting auspice redux state of `controls.geoResolution`.

2. Currently, including `"geographic_info": ["country", "region" ],` in auspice config results in the following in the exported JSON:
```
  "geographic_info": {
    "country": {
      "american_samoa": {
        "latitude": -14.27806,
        "longitude": -170.7025
      },
      "brazil": {
```
I'd also replace this with `geo_resolutions`. And it should be an array of `["country", "region"]` like it is for the config JSON. We may well want to order the dropdown by geographic scale.

3. I believe that `colorings` in both the config JSON and the exported JSON should be arrays rather than dictionaries. It would often be useful to specify order of colorings.

4. ~Currently, `colorings` / `type` accepts three different values `categorical`, `ordinal` and `continuous`. However, I can't think of how auspice would treat `categorical` and `ordinal` values differently. We already "sort" categorical values when a `scale` is provided. If auspice does indeed treat these the same way, I would just replace them both with a single `discrete` value for `type`.~

_Keep current categorical / ordinal behavior._

**Comment by rneher** [comment]
`categorical` and `ordinal` would have different default color scales, right?

**Comment by trvrb** [comment]
>`categorical` and `ordinal` would have different default color scales, right?

This makes sense. I see... At the moment auspice checks for either (https://github.com/nextstrain/auspice/blob/v2/src/util/colorScale.js#L153), but there's a TODO for exactly this. I'll edit my original list to remove this.

**Comment by emmahodcroft** [comment]
I don't have any problem with `"geo_resolutions"`, or changing this to an array so that order can be specified. 
I also don't have issue with making `colorings` an array so we can specify this order. I mentioned this today in #346 [here](https://github.com/nextstrain/augur/pull/346#issuecomment-519906497) in that it might be useful to make our main colorbys like Genotype and Sampling Date be at the top by default - would be easy enough to have this be overridden by order in `config` (or, if not the 'main' colorbys, by order in `--color-by-metadata`. 

If we do these it would need to change in:
- `augur export v2` including depreciation warnings
- Schema for validating v2
- `auspice` in what it expects for these formats/variables
- My help guide thing in docs

**Comment by jameshadfield** [comment]
Closed by #361
